### ISSUE TITLE
pytorch-quantization convert to onnx with acc dropped

### ISSUE THREAD

**Issue by maoxiaoming86** [NVIDIA/TensorRT#1106]
## Description

When I use ./tools/pytorch-quantization/examples/torchvision/classification/classification_flow.py to do QAT for 1epoch，got the followong result.

Accuracy summary:
+------------+-------+
| Stage      |  Top1 |
+------------+-------+
| Initial    | -1.00 |
| Calibrated | -1.00 |
| Finetuned  | 76.31 |
| ONNX       | 75.29 |
+------------+-------+

Why the result of ONNX is dropped about 1.1% ? Is this normal or a bug?

> ## Environment
> **TensorRT Version**: 7.2
> **NVIDIA GPU**: RTX3090
> **NVIDIA Driver Version**: 460.32.03
> **CUDA Version**: 11.0
> **CUDNN Version**: 8.0.5.39
> **Operating System**: CentOS Linux release 7.9.2009 (Core)
> **Python Version (if applicable)**: 3.6.12
> **Tensorflow Version (if applicable)**: 1.15.3
> **PyTorch Version (if applicable)**: 1.7.1
> **Baremetal or Container (if so, version)**:
>

**Comment by ttyio** [comment]
@maoxiaoming86 , you can also try to test the sensitivity of the Q/DQ nodes in your network, and remove some of them to recover the accuracy, here is one sample code that might helps:
https://github.com/NVIDIA/NeMo/blob/main/examples/asr/quantization/speech_to_text_quant_infer.py

check the implementation for `--sensitivity`

**Comment by ttyio** [comment]
Closing since no activity in this thread for more than 3 weeks, please reopen if you still have question, thanks!
